addons/web_editor/static/src/components/html_history_dialog/html_history_dialog.js

```diff
@@ -0,0 +1,115 @@
+/** @odoo-module **/
+
+import { Dialog } from '@web/core/dialog/dialog';
+import { formatDateTime } from '@web/core/l10n/dates';
+import { useService } from "@web/core/utils/hooks";
+import { memoize } from "@web/core/utils/functions";
+import {Component, onMounted, useState, markup} from '@odoo/owl';
+import { _t } from "@web/core/l10n/translation";
+const { DateTime } = luxon;
+
+class HtmlHistoryDialog extends Component {
+    static template = "web_editor.HtmlHistoryDialog";
+    static components = { Dialog };
```

Comment:
Isn't that a little weird ? The component is named SomethingDialog, but instead of _being_ a Dialog, it _has_ a Dialog ? I think it should either extend Dialog or not be named SomethingDialog.

Reply:
That's how OWL work.
ie : https://github.com/odoo/odoo/blob/master/addons/mail/static/src/core/common/message_confirm_dialog.js